app/controllers/concerns/csv_email_action.rb

```diff
@@ -25,7 +25,7 @@ def queue_csv_report_email(report)
     yield_email_and_respond_for_report do |email|
       # TODO: in order to use ActiveJob's `deliver_later`, the report needs to be
       # specifically serializable, which many of our reports are not.
-      CsvReportMailer.delay.csv_report_email(email, report)
+      CsvReportMailer.with(to_address: email, report: report).csv_report_email.deliver_later
```

Comment:
Perhaps this comment was here on purpose, and that's why we should stick with `.delay`? I think that I'd like @giladshanan to weigh in on this.